pkg/helper/client.go

```diff
@@ -261,46 +270,48 @@ func ApplyDesiredState(desiredState nmstatev1alpha1.State) (string, error) {
 		return setOutput, err
 	}
 
+	commitID := readCommitID(setOutput)
+
 	// Future versions of nmstate/NM will support vlan-filtering meanwhile
 	// we have to enforce it at the desiredState bridges and outbound ports
 	// they will be configured with vlan_filtering 1 and all the vlan id range
 	// set
 	bridgesUpWithPorts, err := getBridgesUp(desiredState)
 	if err != nil {
-		return "", rollback(fmt.Errorf("error retrieving up bridges from desired state"))
+		return "", rollback(commitID, fmt.Errorf("error retrieving up bridges from desired state"))
```

Comment:
Let's start to rall Wrapf little by little is the defacto stuff fro future 1.13 but we can start to use it at 1.12 with github.com/pkg/errors

Reply:
That would deserve a refactoring of the module. I don't want to mess too much with this PR.

We should refactor the helper module completely